src/crate/client/doctests/https.txt

```diff
@@ -80,5 +80,5 @@ invalid::
     >>> client.server_infos(crate_host)
     Traceback (most recent call last):
     ...
-    crate.client.exceptions.ConnectionError: Server not available, exception: ...[SSL: ...
+    crate.client.exceptions.ConnectionError: Server not available, exception:...
```

Comment:
Is there anything else in the traceback that would indicate that the certificate is invalid? Asserting only the `ConnectionError` seems a bit generic.

Reply:
On this matter, I just received this error with traceback (from [1]):
```
File "D:\a\crate-python\crate-python\src\crate\client\doctests\https.txt", line 80, in https.txt
Failed example:
    client.server_infos(crate_host)
Expected:
    Traceback (most recent call last):
    ...
    crate.client.exceptions.ConnectionError: Server not available, exception: ...[SSL: ...
Got:
    Traceback (most recent call last):
      File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\doctest.py", line 1337, in __run
        compileflags, 1), test.globs)
      File "<doctest https.txt[11]>", line 1, in <module>
        client.server_infos(crate_host)
      File "D:\a\crate-python\crate-python\src\crate\client\http.py", line 402, in server_infos
        response = self._request('GET', '/', server=server)
      File "D:\a\crate-python\crate-python\src\crate\client\http.py", line 502, in _request
        "Server not available, exception: %s" % ex_message
    crate.client.exceptions.ConnectionError: Server not available, exception: HTTPSConnectionPool(host='localhost', port=65534): Max retries exceeded with url: / (Caused by ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)))
```

[1] https://github.com/crate/crate-python/pull/388/checks?check_run_id=1348073153#step:5:156

Reply:
> unless this error got masked by any other phenomenon

Forget about my last comment. The reason for the `ConnectionResetError` is coming from the fact that CrateDB has not spun up completely, see also https://github.com/crate/crate-python/pull/388#issuecomment-721191227.

Reply:
The problem is, this failing https test doesnt actually set up a cratedb cluster to do it. It just creates a generic https server and attempts to query it with the client, see: https://github.com/crate/crate-python/blob/master/src/crate/client/tests.py#L210

Reply:
I see. Thanks!